src/elements/radioButton/radioButton.spec.js

```diff
@@ -0,0 +1,77 @@
+describe('Semantic-UI: Elements - smRadioButton', function() {
+  'use strict';
+
+  var $scope, $compile, html;
+
+  beforeEach(module('semantic.ui.elements.radioButton'));
+
+  beforeEach(inject(function($rootScope, _$compile_) {
+    $scope = $rootScope.$new();
+    $compile = _$compile_;
+  }));
+
+  function renderRadioBtns($scope) {
+    html = angular.element([
+      '<sm-radio-group ng-model="radio.value">',
+        '<sm-radio-button value="1">One</sm-radio-button>',
+        '<sm-radio-button value="2">Two</sm-radio-button>',
+        '<sm-radio-button value="3">Three</sm-radio-button>',
+      '</sm-radio-group>'].join('')
```

Comment:
something like

```js
function createRadioGroup(model, valuesAndLabels) {
  var template = '<sm-radio-group ng-model="' + model + '">' +
                          Object.getOwnPropertyNames(valuesAndLabels).map(function(value) {
                            var label = valuesAndLabels[value];
                            return '<sm-radio-button value="' + value + '">' + label + '</sm-radio-button>';
                          }).join("") + '</sm-radio-group>';
  element = $compile(template)($scope);
  // and whatever other setup you want to do
}

var kBaseKeys = Object.freeze({
  1: "One",
  2: "Two",
  3: "Three"
});

createRadioGroup("value",  kBaseKeys);
```

or something like that --- but it's your call man, it can just be a bit nicer to write tests without having to do lots of string concatenation and stuff